### ISSUE TITLE
Step "minishift ssh -- less /proc/meminfo" gets stuck on Windows 7

### ISSUE THREAD

**Issue by agajdosi** [minishift/minishift#1990]
For some reason on some Windows 7 machines command `minishift ssh -- less /proc/meminfo` gets stuck on both `cmd` and `PS`. One must then hit the `Q` button to quit the stuck command. Seems there are wrong line endings. Same situation is when used directly inside the Minishift VM.
Log:
```
PS C:\Users\CDKQE> minishift ssh -- 'less /proc/meminfo'
MemTotal:        4046560 kB
MemFree:          558012 kB
MemAvailable:    3047140 kB
Buffers:          115876 kB
Cached:          2555880 kB
SwapCached:            0 kB
Active:           918228 kB
Inactive:        2391272 kB
Active(anon):     638672 kB
Inactive(anon):    16520 kB
Active(file):     279556 kB
Inactive(file):  2374752 kB
Unevictable:           0 kB
Mlocked:               0 kB
SwapTotal:       1023996 kB
SwapFree:        1023996 kB
Dirty:                32 kB
Writeback:             0 kB
AnonPages:        637572 kB
Mapped:           171456 kB
Shmem:             17448 kB
Slab:             111548 kB
SReclaimable:      75904 kB
←[K←[?1l←>←[?1049lPS C:\Users\CDKQE>
```
When `cat` is used, there is no problem, so we can probably just go for cahnging `less` for `cat`.

**Comment by praveenkumar** [comment]
@agajdosi `less` is always going to get stuck and require `q` for quit. I am not sure why you think this should work without interaction. If you think we are using somewhere this in the integration test then please go ahead and change to `cat` without waiting since downstream CI for windows is still failing.

**Comment by agajdosi** [comment]
@praveenkumar I see, did not realized that before. Quite strange however it passes on most machines. Fixing now.

**Comment by LalatenduMohanty** [comment]
Resolved by #1995
